bin/kano-empty-trash

```diff
@@ -42,17 +43,44 @@ def empty_all_trash():
         # it thinks it must be root if sudo is in effect, but this is not the case here.
         os.system('sudo -H -u#{} SUDO_USER={} /usr/bin/kano-empty-trash '.format(uid, uname))
 
+def clean_old_dumps(dir, age):
```

Comment:
Could we change `dir` to something else? It's an internal python command

Reply:
👍